core/query/playtime_entries.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from typing import List, Tuple
+
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm import Session
+
+from api.model.time_tracking import PlaytimeEntriesPost, PlaytimeEntriesPostSummary
+from core.model import create
+from core.model.collection import Collection
+from core.model.identifier import Identifier
+from core.model.library import Library
+from core.model.time_tracking import PlaytimeEntry
+from core.util.datetime_helpers import utc_now
+
+
+class PlaytimeEntries:
+    # The oldest entry acceptable by the insert API
+    # If anything earlier arrives, we ignore it with a 401 response
+    OLDEST_ACCEPTABLE_ENTRY_DAYS = 120
+
+    @classmethod
+    def insert_playtime_entries(
+        cls,
+        _db: Session,
+        identifier: Identifier,
+        collection: Collection,
+        library: Library,
+        data: PlaytimeEntriesPost,
+    ) -> Tuple[List, PlaytimeEntriesPostSummary]:
+        """Insert into the database playtime entries from a request"""
+        responses = []
+        summary = PlaytimeEntriesPostSummary()
+        today = utc_now().date()
+        for entry in data.time_entries:
+            status_code = 201
+            message = "Created"
+            transaction = _db.begin_nested()
+            try:
+                if (
+                    today - entry.during_minute.date()
+                ).days > cls.OLDEST_ACCEPTABLE_ENTRY_DAYS:
+                    # This will count as a success, since we don't want to repeat the entry
+                    status_code = 401
+                    message = "Gone"
```

Comment:
This, too, should be 410. Could we make the message this (or something similarly descriptive)?
```
"Time entry too old and can no longer be processed."
```